packages/shared/src/utils.ts

```diff
@@ -156,6 +156,8 @@ export async function pollForFileData(
     if (json.status !== "done") return null;
 
     await callback?.(json);
+
+    return json;
```

Comment:
😅

Reply:
I wanted this function to simply return final result so that I do not need to pass callback to do so